bin/pgenv

```diff
@@ -183,6 +183,87 @@ pgenv_check_dependencies(){
     fi
 }
 
+# This function provides the specified version from an alias name.
+# WARNING: this does not include beta versions!!!
+#
+# Accepted parameters are either 'latest' or 'oldest' to select
+# the latest stable version installed or the oldest one.
+pgenv_find_version_on_disk(){
+    local search_for=$1
+    local brand
+    local year
+    local selected
+    local patch
+    local current_version
+    local selected_value
+
+    # find sed even without the configuration/dependencies
+    if [ -z "$PGENV_SED" ]; then
+        PGENV_SED=$(which sed)
+    fi
+
+    for version in $( ls -d pgsql-* 2>/dev/null | $PGENV_SED 's/pgsql-//' ); do
+        pgenv_debug "Version $version"
+        brand=$( echo $version | $PGENV_SED 's/\([[:digit:]]\{1,2\}\).*/\1/' )
+        year=$(  echo $version | $PGENV_SED 's/[[:digit:]]\{1,2\}\.\([[:digit:]]\{1,2\}\).*/\1/' )
+        patch=$(  echo $version | $PGENV_SED 's/[[:digit:]]\{1,2\}\.[[:digit:]]\{1,2\}\.\{0,1\}\([[:digit:]]\{0,2\}\).*/\1/' )
+
+        current_version="${brand}${year}${patch}"
+        pgenv_debug "Found version [$current_version]"
+
+        case "$search_for" in
+            latest)
+                if [ -z "$selected_value" ] || [ $selected_value -lt $current_version ]; then
+                    selected_value=$current_version
+                    selected=$version
+                fi
+                ;;
+            oldest)
+                if [ -z "$selected_value" ] || [ $selected_value -gt $current_version ]; then
+                    selected_value=$current_version
+                    selected=$version
+                fi
+                ;;
+        esac
+    done
+
+    pgenv_debug "Version selected for [$search_for] = $selected"
+    echo $selected
+}
+
+
+# Guesses the selected PostgreSQL version to use.
+# The idea is that every time a command needs a PostgreSQL version number
+# this function should be called.
+# As an example:
+# v=$( pgenv_guess_postgresql_version $2 )
+#
+# - with $2 = 13.0 will return it (13.0);
```

Comment:
Mitght be cool to return the latest version for a major number. For example, if '13.0` and `13.1` are both installed, for `13`, return `13.1`.